src/index.ts

```diff
@@ -2,19 +2,20 @@
 export * from './app'
 export * from './app-client'
 export * from './app-deploy'
-export * from './asset'
 export * from './debugging'
-export * from './indexer-lookup'
 export * from './transaction'
-export * from './transfer'
 
 // Legacy exports - remove after 2 x major version bumps
 export * from './account'
+export * from './asset'
 export * from './dispenser-client'
+export * from './indexer-lookup'
 export * from './localnet'
 export * from './network-client'
+export * from './transfer'
 
 // Up to date exports
 export * from './amount'
 export * from './config'
+export * as indexer from './indexer-lookup'
```

Comment:
I think the indexer methods are useful as-is. Per previous discussions we don't want to make them that easy to use given preference to discourage use of indexer. I figure exposing them through an `indexer` export off the `/` is a good way of exposing them without making for a noisy `/` import.